tests/unit/transforms/json-string-test.js

```diff
@@ -0,0 +1,27 @@
+import { expect } from 'chai';
+import { describeModule, it } from 'ember-mocha';
+import Post from 'ghost-admin/models/post';
+
+describeModule(
+    'transform:json-string',
+    'Unit | Transform | json string',
```

Comment:
Very minor thing but could this be updated to match the naming scheme of the other tests? `Unit: Transform: json-string` (it helps to be consistent so that test filters pick up everything that's expected)